app/views/partials/vat/card/panel_info.scala.html

```diff
@@ -35,68 +35,19 @@
         </div>
     </div>
 
-    <div id="vat-card-panel-info2" class="panel panel-border-wide">
-            <strong class="bold-small">@messages("bt.vat.card.delay.period_ended")</strong>
-        <div style="margin-top: 20px">
-            <p>@messages("bt.vat.card.delay.deferred") <a id="vat-delayed-link" target="_blank" data-journey-click="link - click:VAT cards:VAT deferral new payment scheme" href="https://www.gov.uk/guidance/deferral-of-vat-payments-due-to-coronavirus-covid-19">@messages("bt.vat.card.delay.link")</a>.</p>
-            <p>@messages("bt.vat.card.full_amount")</p>
-        </div>
-    </div>
     } else {
```

Comment:
I'd remove everything after and including "else {". It would be good to remove the argument "eligibleforPlan" too, but that might take too much time for this task.